src/components/SuiteHeader/SuiteHeader.story.jsx

```diff
@@ -227,6 +228,99 @@ export const HeaderWithSurveyNotification = () => {
   );
 };
 
+/* Sample of SuiteHeader usage with data hook
+export const HeaderWithHook = () => {
+  const StatefulExample = () => {
+    const [data, isLoading, error, refreshData] = useSuiteHeaderData({
+      // baseApiUrl: 'http://localhost:3001/internal',
+      domain: 'mydomain.com',
+      isTest: true,
+      surveyConfig: {
+        id: 'suite',
+        delayIntervalDays: 30,
+        frequencyDays: 90,
+      },
+      lang: 'en',
+    });
+    const surveyData = data.showSurvey ? { surveyLink: 'https://www.ibm.com', privacyLink: 'https://www.ibm.com'} : null;
+    return (
+      <SuiteHeader
+        suiteName="Application Suite"
+        appName="Application Name"
+        userDisplayName={data.userDisplayName}
+        username={data.username}
+        routes={data.routes}
+        applications={data.applications}
+        i18n={data.i18n}
+        surveyData={surveyData}
+      />
+    );
+  }
+  return <StatefulExample />;
+};
+
+HeaderWithHook.story = {
+  name: 'Header with hook',
+};
```

Comment:
Is there something wrong with these stories if they're uncommented and included in storybook?

Reply:
@tay1orjones Yes, these stories are testing async data loading, which is not the scope of regular component stories, it is just to make sure that the data hook is working as expected during development (the data hook is excluded from coverage as well). Besides, these stories mount the component without data in the first render, which would make the stories fail as some properties are required. Also, for some reason, stories fail for containing async tasks, with this error:
```
Failed: "Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in %s.%s"
```
If you prefer, I can just delete the stories instead of commenting them out.

Reply:
I'm fine either way. If it's helpful in development I think we could leave them uncommented, but if the stories aren't helpful in development and also fail, then we can remove them.